formats/e2hs.md

```diff
@@ -0,0 +1,41 @@
+# E2HS files
+
+ E2HS files are themselves e2store files. For more information on this format,
+ see the [documentation](https:github.com/status-im/nimbus-eth2/blob/stable/docs/e2store.md).
+
+ The overall structure of an E2HS file follows closely the structure of an Era file
+ which contains consensus Layer data (and as a byproduct, EL data after the merge).
+
+ The structure can be summarized through this definition:
+
+	e2hs := Version | block-tuple* | other-entries* | BlockIndex
+	block-tuple :=  CompressedHeaderWithProof | CompressedBody | CompressedReceipts
+
+ Each basic element is its own entry:
+
+	Version                     = { type: 0x6532, data: nil }
```

Comment:
the type is a byte array ie better stick to the upstream array notation [`[0x65, 0x32]`](https://github.com/status-im/nimbus-eth2/blob/stable/docs/e2store.md#version) to avoid potential byte order issues.